packages/terra-framework-docs/src/terra-dev-site/doc/slider/Examples.2/CustomSlider.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import Slider from 'terra-slider';
+
+const CustomSlider = () => (
+  <Slider minimumValue={0} maximumValue={100} labelName="Field Label" minimumLabel="0%" maximumLabel="100%" defaultValue={20} />
```

Comment:
We should avoid using dummy values like "Label" in our examples. Something like "Zoom", "Size", "Intensity" or something more natural like that would be better.

Reply:
Updated.